sample_scripts/third_party_platform_exporting/magento/v1.9/soap_api/orders.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+require 'savon'
+require 'json'
+
+require_relative 'transporter_exporter'
+
+class OrderExporter < TransporterExporter
+  def run
+    orders_for(required_env_vars['MAGENTO_STORE_ID']).each do |order|
+      next if skip?(order)
+
+      puts JSON.pretty_generate(order.merge('items' => items_for(order)))
```

Comment:
Do we need to add any rescues here and for customers above in case we fail to retrieve additional details for certain orders/customers?

Reply:
It's pretty similar to how you would call `.variants` on a `ShopifyAPI::Product` object.  There's no connection state across requests, so I don't foresee how it could blow up unless the product was maybe deleted during the execution of the export...?

Reply:
@ariens-shopify yes.

I'm worried about the soap call for these causing an exception that could fail the whole process/ we won't be able to recover from issues retrieving the rest of the orders if a single one fails. 

However, I'm content with adding an issue for this and dealing with this in the future - we don't have a recovery strategy if retrieving an order fails (ex: do we store the ids of all the orders and output them at the end, or do we retry, etc.)